packages/apputils-extension/schema/sessions-settings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Terminals and Kernels sessions settings",
+  "description": "Terminals and Kernels sessions settings",
+  "additionalProperties": false,
+  "properties": {
+    "showStatusBarItem": {
+      "type": "boolean",
+      "title": "Show the status bar item",
+      "description": "Whether to show the running terminals and kernels item in the status bar",
+      "default": true
+    },
```

Comment:
Maybe this should be just "Kernel settings" and later we can also fold this orphan in there:

![image](https://github.com/user-attachments/assets/5f7ae75d-6639-45e3-899e-d2763d1b7d61)


My reasoning is:
- the terminals are not shown in status bar unless any is open
- we can rework the behaviour of `showStatusBarItem` to only affect kernels
- we could add another `showStatusBarItem` for terminal specifically, as we already have "Terminal" settings section:

![image](https://github.com/user-attachments/assets/8392c99b-2faf-4193-be0e-adc299c3ae40)

Basically I would like to decouple these so that we do not have "terminals and kernels" section in settings just because the "running sessions" encompasses both in code.

CC @afshin @jtpio?

Reply:
> I would like to decouple these so that we do not have "terminals and kernels" section in settings just because the "running sessions" encompasses both in code.

To make it clear, I do not suggest making this change in this PR. I am only asking if this is the direction we should take (so in this PR we would just rename this settings section to "Kernels", and possibly the file to `kernel.json`).

Reply:
I opened https://github.com/jupyterlab/jupyterlab/issues/17406 to track this.